queries/Query1.sql

```diff
@@ -0,0 +1,29 @@
+-- List of Borrowed Books: Retrieve all books borrowed by a specific borrower, 
+-- including those currently unreturned.
+
+USE [LibraryManagement]
+GO
+
+-- Specific borrower ID 
+DECLARE @BorrowerID NUMERIC(18, 0) = 5;
```

Comment:
You can use INT instead of numeric for better efficiency -> DECLARE @BorrowerID INT = 5;